src/user_prompts.rs

```diff
@@ -0,0 +1,39 @@
+//! User prompt related functionality for WebDriver.
+//!
+//! See [18. User Prompts](https://www.w3.org/TR/webdriver1/#user-prompts) of the WebDriver
+//! standard.
+use webdriver::command::WebDriverCommand;
+
+use crate::error;
+use crate::Client;
+
+/// `PromptAction` enumerates the different actions a `Client` can take in response to a user prompt
+/// or alert in the browser window.
+#[derive(Debug, Clone)]
+#[non_exhaustive]
+pub enum PromptAction {
+    /// `Accept` is equivalent to a user clicking the `OK` button in the prompt.
+    Accept,
+    /// `Dismiss` is equivalent to a user clicking the `Cancel` or `OK` button in the prompt,
+    /// whichever is present and appears first.
```

Comment:
I don't think this is _quite_ accurate. The wording from the spec is:
> To dismiss the current user prompt, do so as if the user would click the Cancel or OK button, whichever is present, in that order. 

Note that it's not "which appears first", but "in that order", which is an important distinction. With the phrasing you're currently using, a user might think that if the OK button appears to the left of the Cancel button, it'd be clicked.

Also, let's remove the backticks around Cancel and OK here and above — they're not code-related. I'd go with `_` for italics instead.

Reply:
Ha - good catch, I misunderstood the spec when reading it. I'll update it now.